### ISSUE TITLE
state-rent example test failure

### ISSUE THREAD

**Issue by shunjizhan** [AcalaNetwork/bodhi.js#149]
```
 0 passing (9s)
  1 failing

  1) StateRent
       stateRent works:
     Error: -32603: execution fatal: Module { index: 180, error: 2, message: Some("NoPermission") }
      at RpcCoder._checkError (/Users/shunjizhan/Acala/bodhi.js/common/temp/node_modules/.pnpm/@polkadot+rpc-provider@6.9.2/node_modules/@polkadot/rpc-provider/coder/index.cjs:84:13)
      at RpcCoder.decodeResponse (/Users/shunjizhan/Acala/bodhi.js/common/temp/node_modules/.pnpm/@polkadot+rpc-provider@6.9.2/node_modules/@polkadot/rpc-provider/coder/index.cjs:47:10)
      at WsProvider.value (/Users/shunjizhan/Acala/bodhi.js/common/temp/node_modules/.pnpm/@polkadot+rpc-provider@6.9.2/node_modules/@polkadot/rpc-provider/ws/index.cjs:260:90)
      at W3CWebSocket.value [as onmessage] (/Users/shunjizhan/Acala/bodhi.js/common/temp/node_modules/.pnpm/@polkadot+rpc-provider@6.9.2/node_modules/@polkadot/rpc-provider/ws/index.cjs:240:153)
      at W3CWebSocket._dispatchEvent [as dispatchEvent] (/Users/shunjizhan/Acala/bodhi.js/common/temp/node_modules/.pnpm/yaeti@0.0.6/node_modules/yaeti/lib/EventTarget.js:107:17)
      at W3CWebSocket.onMessage (/Users/shunjizhan/Acala/bodhi.js/common/temp/node_modules/.pnpm/websocket@1.0.34/node_modules/websocket/lib/W3CWebSocket.js:234:14)
      at WebSocketConnection.<anonymous> (/Users/shunjizhan/Acala/bodhi.js/common/temp/node_modules/.pnpm/websocket@1.0.34/node_modules/websocket/lib/W3CWebSocket.js:205:19)
      at WebSocketConnection.EventEmitter.emit (domain.js:467:12)
      at WebSocketConnection.processFrame (/Users/shunjizhan/Acala/bodhi.js/common/temp/node_modules/.pnpm/websocket@1.0.34/node_modules/websocket/lib/WebSocketConnection.js:554:26)
      at /Users/shunjizhan/Acala/bodhi.js/common/temp/node_modules/.pnpm/websocket@1.0.34/node_modules/websocket/lib/WebSocketConnection.js:323:40
      at processTicksAndRejections (internal/process/task_queues.js:75:11)
```

**Comment by shunjizhan** [comment]
full logs
<img width="1508" alt="Screen Shot 2021-11-26 at 10 18 40" src="https://user-images.githubusercontent.com/16548786/143516968-7b85dab5-0499-4c60-a6d7-0e54fcfe5f68.png">

**Comment by shunjizhan** [comment]
fixed by #151
